### DISCUSSION TITLE
blog/angular-control-flow

### DISCUSSION THREAD

**Discussion by giscus** [angular-experts-io/angular-experts-website-comments#18] [General]
# blog/angular-control-flow

A new modern way of writing ifs, if-else, switch-cases, and even loops in Angular templates!

https://angularexperts.io/blog/angular-control-flow

**Comment by raphael22** [comment]
Hi !
Thanks for the clear and concise recap.
You have a typo in your empty block exemple though, you use @placeholder instead.
Cheers.
